File: BackEnd_ExpressJS/app/models/language.js
const { DataTypes } = require('sequelize');
const sequelize = require('../config/database');
const Translation = require('./translation');

const Language = sequelize.define('tblLanguage', {
        ideLanguage: {
            type: DataTypes.STRING,
            primaryKey: true,
        },
        strLanguage: {
            type: DataTypes.STRING,
            allowNull: false,
        },
        ideTranslation: {
            type: DataTypes.BIGINT,
            allowNull: true,
            references: {
              model: Translation,
              key: 'ideTranslation',
            },
            onUpdate: 'CASCADE',
            onDelete: 'SET NULL',
        },
        createdAt: {
            type: DataTypes.DATE,
            allowNull: false,
        },
        updatedAt: {
            type: DataTypes.DATE,
            allowNull: false,
        },
    }, {
        // Activar/Desactivar las marcas de tiempo
        timestamps: true,
        // Usar el nombre de la tabla tal como está en la base de datos
        freezeTableName: true,
    });

Language.belongsTo(Translation, { foreignKey: 'ideTranslation' });

module.exports = Language;

File: BackEnd_Django/api/myModels/language.py
from django.db import models
from .translation import Translation

class Language(models.Model):
    ideLanguage = models.CharField(max_length=255, primary_key=True)
    strLanguage = models.CharField(max_length=255)
    ideTranslation = models.ForeignKey(Translation, on_delete=models.SET_NULL, null=True, blank=True)
    createdAt = models.DateTimeField(auto_now_add=True)
    updatedAt = models.DateTimeField(auto_now=True)

    class Meta:
        db_table = 'tblLanguage'

    def __str__(self):
        return self.strLanguage
